--- a/arch/arm/include/asm/arch-mx6/opos6ul.h
+++ b/arch/arm/include/asm/arch-mx6/opos6ul.h
@@ -6,6 +6,7 @@
 #ifndef __ARCH_ARM_MX6UL_OPOS6UL_H__
 #define __ARCH_ARM_MX6UL_OPOS6UL_H__
 
+int opos6ul_board_init(void);
 int opos6ul_board_late_init(void);
 
 #endif

--- a/arch/arm/mach-imx/mx6/opos6ul.c
+++ b/arch/arm/mach-imx/mx6/opos6ul.c
@@ -107,12 +107,17 @@
 }
 #endif /* CONFIG_FEC_MXC */
 
+int __weak opos6ul_board_init(void)
+{
+       return 0;
+}
+
 int board_init(void)
 {
 	/* Address of boot parameters */
 	gd->bd->bi_boot_params = CONFIG_SYS_SDRAM_BASE + 0x100;
 
-	return 0;
+	return opos6ul_board_init();
 }
 
 int __weak opos6ul_board_late_init(void)
