--- a/include/configs/opos6uldev.h
+++ b/include/configs/opos6uldev.h
@@ -109,7 +109,7 @@
 	"mmcroot=/dev/mmcblk0p2 ro\0"                                                           \
 	"mmcrootfstype=ext4 rootwait\0"                                                         \
 	"kernelimg="           __stringify(CONFIG_BOARD_NAME)          "-linux.bin\0"           \
-	"videomode=video=ctfb:x:800,y:480,depth:18,pclk:33033,le:96,ri:96,up:20,lo:21,hs:64,vs:4,sync:0,vmode:0\0" \
+	"videomode=video=ctfb:x:800,y:480,depth:18,pclk:30030,le:30,ri:210,up:23,lo:22,hs:16,vs:1,sync:1073741824,vmode:0\0" \
 	"check_env=if test -n ${flash_env_version}; "                                           \
 		"then env default env_version; "                                                \
 		"else env set flash_env_version ${env_version}; env save; "                     \
